packages/circuit-ui/components/Body/Body.mdx

```diff
@@ -22,6 +22,12 @@ The Body component comes in three sizes. Use the default `m` size in most cases.
 
 <Story of={Stories.Sizes} />
 
+### Weights
+
+The Body component comes in three weights. Use the default `regular` weight in most cases.
```

Comment:
`The Body component comes in three weights.`
but there is only 2 in the component 
`const weights = ['regular', 'bold'] as const;`
 ?